Blish HUD/Controls/Container.cs

```diff
@@ -118,6 +118,60 @@ public abstract class Container : Control, IEnumerable<Control> {
             set => SetProperty(ref _autoSizePadding, value);
         }
 
+        protected Scrollbar _panelScrollbar;
```

Comment:
Anything for the scrollbar itself should be in the `Panel` control.  Panel is the highest level control that has implemented the scrollbar and I don't see a reason to move its implementation up into the Container.  Containers are intended to be fairly barebones.

If an implementer wishes to share this functionality then they can implement the `Panel` control as they do now.